s2p/cli.py

```diff
@@ -15,11 +15,14 @@ def main():
                         help=('path to a json file containing the paths to '
                               'input and output files and the algorithm '
                               'parameters'))
+    parser.add_argument('--start_from', dest='start_from', type=int,
+                        default=0, help="Restart the process from a given step in "
+                                        "case of an interruption or to try different parameters.")
```

Comment:
Maybe mention it should be the int id of the step in question?

Reply:
Sure, you think this is better?